happi/client.py

```diff
@@ -801,10 +801,32 @@ def from_config(cls, cfg=None):
         cfg_parser = configparser.ConfigParser()
         cfg_file = cfg_parser.read(cfg)
```

Comment:
A bit of a left field suggestion/idea: any thoughts on if we should be able to include paths to other HAPPI_CFG files within our happi config file? For example:

my_happi_config.cfg:

```
[MINE]
path=my_happi_cfg.json
[XPP]
cfg=xpp_config_file.cfg
```

xpp_config_file.cfg, in pink mode:

```
[XPP_HUTCH]
xpp_hutch_pink.json
[XPP_BEAMLINE]
xpp_beamline.json
```

and in mono mode, the hutch changes it to:

```
[XPP_HUTCH]
xpp_hutch_mono.json
[XPP_BEAMLINE]
xpp_beamline.json
```

Reply:
Neat idea

Reply:
I'd personally want to keep everything in one HAPPI_CFG file, so I don't have to go searching for the other files being referenced.  But aside from that I don't really see an issue with this.

Reply:
I guess the idea here was that XPP wouldn't need to change any global files to make other people's lightpath apps learn that the devices are all in the main line now

Reply:
Another way to accomplish the above is to just have XPP change xpp_devices.json as needed